private_set_intersection/javascript/bindings/utils.h

```diff
@@ -34,6 +34,30 @@ emscripten::val ToJSObject(private_join_and_compute::StatusOr<T> statusor) {
   return result;
 }
 
+template <typename T>
+emscripten::val ToSerializedJSObject(
+    private_join_and_compute::StatusOr<T> statusor) {
+  auto result = emscripten::val::object();
+  if (statusor.ok()) {
+    const T protobuf = statusor.ValueOrDie();
+    std::string byte_string;
+    protobuf.SerializeToString(&byte_string);
```

Comment:
You can avoid a copy by first allocating the std::vector<uint8_t> of the correct size and then using SerializeToArray to write to it directly.

Reply:
I tried to implement your suggestion, but I don't think `SerializeToArray` doesn't accept a `vector` type.

Reply:
You can get a raw pointer from a vector by calling vector.data().